src/algorithms/calorimetry/CalorimeterHitDigi.cc

```diff
@@ -20,6 +20,7 @@
 #include <Evaluator/DD4hepUnits.h>
 #include <edm4hep/CaloHitContributionCollection.h>
 #include <fmt/core.h>
+#include <gsl/pointers>
```

Comment:
Odd that this is needed. Looking at the code, I assume it comes from
https://github.com/eic/algorithms/blob/77e78012a79e294fae39091bdee39ac8092f1472/core/include/algorithms/type_traits.h#L10

Reply:
In iwyu we trust.